src/components/secrets/SecretListItem/Actions.js

```diff
@@ -35,6 +35,16 @@ function SecretListItemActions({ secret }) {
         <Dropdown.MenuItem divider />
 
         <Dropdown.MenuItem
+          onSelect={() => MetadataActions.removeSecretFromCurrentFolder({
+            secret,
+            currentFolderId: parentFolderId,
+          })}
+          disabled={!secret.canBeUpdatedBy(currentUser) || typeof parentFolderId === 'undefined'}
```

Comment:
`parentFolderId` may be `null`, `undefined`, or a string. Probably useless to check the type here?